test/docfx.Test/lib/GitHubUserCacheTest.cs

```diff
@@ -0,0 +1,218 @@
+using Newtonsoft.Json.Linq;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Docs.Build
+{
+    public class GitHubUserCacheTest
+    {
+        [Fact]
+        public void GetUserByLogin()
+        {
+            VerifyCore(
+                async (cache) => await cache.GetByLogin("alice"),
+                "{}",
+                "{\"users\":[{\"id\":1,\"login\":\"alice\",\"name\":\"Alice\",\"emails\":[\"alice@contoso.com\"]}]}",
```

Comment:
there is a trick to use single quote and the `Replace("\'", "\")"` for more compact JSON in C#